File: 10 PrimeandComposite/MinPerimeterRectangle/MPR.js
function solution(N) {
  let i = 1;
  let perimeter;

  while ((i * i) <= N) {
    if (N % i === 0) {
      perimeter = 2 * (i + N / i);
    }
    i += 1;
  }
  return perimeter;
}
solution(30);
// O(sqrt(N))

File: Lessons/10 PrimeandComposite/MinPerimeterRectangle/MPR.cpp
int solution(int N) {
  int i = 1;
  int perimeter = 0;

  while ((i * i) <= N) {
    if (N % i == 0) {
      perimeter = 2 * (i + N / i);
    }
    i += 1;
  }
  return perimeter;
}
